src/adapters/User/index.test.ts

```diff
@@ -0,0 +1,25 @@
+import { getAuth } from 'adapters/BaseAuth';
+
+import { getUserInfo } from '.';
+
+jest.mock('adapters/BaseAuth');
+
+describe('UserAdapter', () => {
+  beforeEach(() => {
+    (getAuth as jest.Mock).mockImplementation(() => jest.fn());
```

Comment:
IMO I don't think we need to mock this function

Reply:
@nvminhtue thanks, I removed it in [dc2b9c2](https://github.com/manh-t/react-survey/pull/59/commits/dc2b9c2910ddc2065bd102c378760f03848fdadd)